dotnet/src/Connectors/Connectors.Onnx/Text/OnnxRuntimeGenAIPromptExecutionSettingsJsonSerializerContext - Copy.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Microsoft.SemanticKernel.Connectors.Onnx;
+
+namespace Microsoft.SemanticKernel.Text;
+
+[JsonSerializable(typeof(OnnxRuntimeGenAIPromptExecutionSettings))]
+internal sealed partial class OnnxRuntimeGenAIPromptExecutionSettingsJsonSerializerContext : JsonSerializerContext
```

Comment:
The filename ends in `- Copy.cs`, which I assume is inadvertent.

Reply:
Oops, the file name is log so I did not noticed the suffix :) Fixed.